lib/modules/manager/helmv3/artifacts.ts

```diff
@@ -218,3 +221,12 @@ export async function updateArtifacts({
     ];
   }
 }
+
+function isHelmLockChanged(oldContent: string, newContent: string): boolean {
+  const regex = regEx(/^generated: ".+"$/m);
+  const replacement = 'generated: "stubbed"';
+  return (
+    newContent.replace(regex, replacement) !==
+    oldContent.replace(regex, replacement)
```

Comment:
```suggestion
    newContent.replace(regex, '') !==
    oldContent.replace(regex, '')
```
just replace with empty string